Resources/Prototypes/Entities/Structures/Machines/Computers/computers.yml

```diff
@@ -789,23 +775,19 @@
     interfaces:
       enum.CargoConsoleUiKey.Orders:
         type: CargoOrderConsoleBoundUserInterface
-  - type: LitOnPowered
+  - type: Computer
+    board: CargoRequestComputerCircuitboard
```

Comment:
Cant this just be "Null" to make sure there isnt a way to get it by some weird bug?

Reply:
Done and commented.